android/src/main/res/values/dates.xml

```diff
@@ -0,0 +1,27 @@
+<resources>
+    <string-array name="year_array">
+        <item>Year</item>
+        <item>2016</item>
+        <item>2017</item>
+        <item>2018</item>
+        <item>2019</item>
+        <item>2020</item>
```

Comment:
maybe better to add few more years just to be sure that we cover by default all possible values for near future.